### DISCUSSION TITLE
[Config Support]: Can't Debug Audio or Logs for Specific Camera

### DISCUSSION THREAD

**Discussion by moogoos** [blakeblackshear/frigate#12797] [Config Support]
### Describe the problem you are having

I am debugging an audio event not happening when I expect it to. i was helped in configuring the logging to enable audio via another support thread here: https://github.com/blakeblackshear/frigate/discussions/12775

Now that I have enabled event debug for audio I get things like this:

```
debug
2024-08-06 21:40:49 frigate.events.audio Heard mechanisms with a score of 0.66796875
debug
2024-08-06 21:40:49 frigate.events.audio Heard tick with a score of 0.66796875
debug
2024-08-06 21:40:55 frigate.events.audio Heard speech with a score of 0.5
```

- I dont seem to have the ability to isolate debug for a single camera or stream, which would be nice.
- currently "Heard speech with a score of 0.5" I have no idea which camera is reporting this as well. So it's hard to debug.

Any ideas or options to isolate? If not id like to make a feature request so that:

- logging can be enabled per camera
- log lines define which camera is reporting this event in debug so can atleast filter to the camera where troubleshooting is occuring.

### Version

.14 RC2

### Frigate config file

```yaml
logger:
  default: info
  logs:
    frigate.events.audio: debug
```

### Relevant log output

```shell
2024-08-06 21:40:49 frigate.events.audio Heard mechanisms with a score of 0.66796875
debug
2024-08-06 21:40:49 frigate.events.audio Heard tick with a score of 0.66796875
debug
2024-08-06 21:40:55 frigate.events.audio Heard speech with a score of 0.5
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

Coral

**Comment by NickM-27** [comment]
best way for now is to temporarily disable audio detections on the cameras that are not being debugged. Camera specific logging would be very complicated to implement, definitely makes sense to add the camera name to that log line

**Reply by moogoos** [reply]
kk. added feature request: https://github.com/blakeblackshear/frigate/discussions/12798

**Reply by NickM-27** [reply]
to be clear that is just a discussion, feature requests are added as issues https://github.com/blakeblackshear/frigate/issues/new/choose

in any case not a big deal because I am about to have a PR up for it

**Reply by moogoos** [reply]
yeah i thougth I added it to the right place as "ideas"
